.github/workflows/ci.yml

```diff
@@ -13,17 +13,17 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v2
       with:
-        node-version: '10.14'
+        node-version: '12'
     - name: Cache Node.js modules
       uses: actions/cache@v2
       with:
         path: ~/.npm
-        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/yarn.lock') }}
         restore-keys: |
             ${{ runner.os }}-node-
-    - run: npm ci
+    - run: yarn install
```

Comment:
What are the consequences of switching to yarn? I think we'd want to keep the CI of all repos as uniform as possible and stick to npm. Not only when thinking about mono-repo, but also release automation. Switching the package manger because a dependency cannot be resolved sounds like buying a new car because a light bulb needs to be changed. This is something we should discuss further.

Reply:
> the reason I switched to yarn is that npm just failed to resolve all the dependencies for some reason. After struggling with it I gave up and switched to yarn.

@romanbsd Thanks for clarifying. We are moving towards release automation, which will be focused on npm, so I suggest we stick with npm for now and try to resolve the issue. Could you please give more details about what failed?

Reply:
You can run `npm i` on this branch and see for yourself, the errors are pretty similar between different npm versions. I think it boils down to npm being unable to install version which has a conflicting dependency. I gave it some time, but couldn't fix it without resorting to creating and manually modifying npm-shrinkwrap.json manually.

Reply:
Interesting, are you talking about the current master before this PR?

Reply:
Actually, if I'm not mistaken, I even had problem on master. But here I refer specifically to my branch and my package.json

Reply:
I assume you are talking about the graphql dependency error? Maybe you can revert this PR back to npm and we look at the errors here?

Reply:
`npm i` fails, so it's impossible to generate the package.lock ...

Reply:
interestingly enough, I just tried `npm i` and it worked. Switching back to npm.